### DISCUSSION TITLE
Feature Request: Submodule View for Registry

### DISCUSSION THREAD

**Discussion by jderieg** [terrakube-io/terrakube#470] [Ideas]
We have a Module repo in github that contains several submodules. In terraform cloud this shows up as a list of submodules in the registry that you can click into, as shown below. It would be great to have this feature added to Terrakube!

![tfmodule2](https://github.com/AzBuilder/terrakube/assets/63372114/0e3188c4-8bbc-4126-83a1-b2ffcbabf06a)

**Comment by alfespa17** [comment]
This looks interesting but we will have to check the terraform CLI code, because the use of submodules it is not explained in the "[terraform registry protocol](https://developer.hashicorp.com/terraform/internals/module-registry-protocol)".

**Reply by alfespa17** [reply]
By the way you could create the feature request in this link https://github.com/AzBuilder/terrakube/issues/new/choose, I will close this discusion.
